### DISCUSSION TITLE
Add collection slug to page for easier CSS

### DISCUSSION THREAD

**Discussion by thgh** [payloadcms/payload#1390] [Feature Requests & Ideas]
Some collections have long titles, others short ones. I would like to write a simple CSS rule to style the title appropriately.

Wordpress has several classnames on the body. That works!

As I was writing this, I made a proof-of-concept: [payload-plugin-body-classname](https://www.npmjs.com/package/payload-plugin-body-classname)

**Comment by peiris** [comment]
@thgh Thank you for POC. was looking for exactly this. Am surprised this isn't a default 🤦🏻‍♂️
